### PR TITLE
Update Event Map Styles

### PR DESCRIPTION
**Pull request by iandunn**
See #3
Builds on #40, so this shouldn't be merged before that is. Only the map commit needs to be reviewed.

Updates the colors, icons, etc used in the map displayed on the What's On page.

Screenshot 2023-04-05 at 4 22 05 PM

Screenshot 2023-04-05 at 4 22 31 PM

### CODE REVIEW

**Comment by iandunn** [comment]
This is still a WIP. I broke it off of #40 to try and keep that PR smaller. I'll circle back and finish this once #40 is done.

File: wp-content/themes/twentyseventeen-wp20/style.css [hunk 1]
```diff
@@ -903,6 +903,7 @@ body.page-slug-whats-on h1.entry-title {
 #wp20-events-map {
 	background-color: #AADAFF;
 	margin: 0 auto 40px;
+	height: 508px;
```
Comment (adamwoodnz) [inline wp-content/themes/twentyseventeen-wp20/style.css]: Figma shows this as 490px high on mobile and 530px high on desktop

| Mobile | Desktop |
|-|-|
| Screen Shot 2023-04-06 at 2 37 05 PM | Screen Shot 2023-04-06 at 2 37 34 PM |

**Review by adamwoodnz — COMMENTED** [review]
This looks really nice, just the height is out I think.

On a side note it would be cool if the events were filter based on map zoom 😁

File: wp-content/themes/twentyseventeen-wp20/style.css (hunk 1 above)
Comment (iandunn) [reply wp-content/themes/twentyseventeen-wp20/style.css]: Huh, you're right. I remember seeing `508px` somewhere, but I don't see it now. 

Fixed in 7868dd1, thanks!

**Comment by marko-srb** [comment]
Screen Shot 2023-04-07 at 13 42 08

@iandunn 

Not sure the screen size your screenshot comes from, but there are several things that seem off (text size, button search size and so on). See above screenshot. On the other hand, regarding the map only: 

1. Height. Map should be less height, so it consumes less space. See design.
2. Mobile. Is there an option to move Google logo just a bit up so it isn't overlayed with info. This isn't big of a deal, but if possible it would be neat. 
3. Due to the comment on top and screenshot, I'd need to see this live. 

Regardless of those, overall style is on point and looks good. 

Thanks.

**Comment by iandunn** [comment]
Thanks! 

> text size, button search size and so on

Those may already be addressed in #40. This PR just focuses on the map, so #40 is the best place for any other feedback.

> Map should be less height

I see `530px` for the desktop map in Figma, and `490px` for mobile. Those are the values that I used, but let me know if you intended it to be more dynamic or something. We could add something like `max-height: 60vh` if you want it to never take up more than a certain percentage of the viewport. 

> move Google logo just a bit up so it isn't overlayed with info

Good catch! I fixed that in f19dc60.

> I'd need to see this live.

I'll see if I can get all the related PRs merged today and pushed out to the staging site 👍🏻 We can keep iterating after that.

**Pull request merged**